main/static/index.html

```diff
@@ -71,9 +71,15 @@
 
     <div>
       <div class="col-xs-12">
-        <table ng-show="isTabular()" class="table table-striped">
+        <table ng-show="isTabular() && queryResult.name != 'describe'" class="table table-striped">
           <tr ng-repeat="row in queryResult.body"><td>{{row}}</td></tr>
         </table>
+        <div ng-show="isTabular() && queryResult.name == 'describe'">
+          <table ng-repeat="(key, list) in queryResult.body" class="table table-striped table-column">
```

Comment:
can we transform the data, so that we can display them in

```
|A | B | C|
|---------|
|a1| b1| c1|
|a2| b2| c2|
|a3| b3| c3|
```
?

we can do the transformation on the client side.